docs/source/developer_resources/Developing_Apps.rst

```diff
@@ -0,0 +1,45 @@
+
+Supported Application Types
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- Puthon
+- Java (Jar)
+
+Registering Application With Platform
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Platform requires a config file for each application on a particular location in order to register the applications.
+
```

Comment:
The GridAPPS-D platform requires a configuration file for ...  specify the location rather than "a particular" location.

Reply:
Done